torch_xla/csrc/data_ops.cpp

```diff
@@ -100,6 +100,17 @@ xla::XlaOp SqueezeAllTrivialDimensions(xla::XlaOp input) {
   return XlaHelpers::DynamicReshape(input, output_sizes);
 }
 
+xla::XlaOp BuildDynamic(xla::XlaOp input,
```

Comment:
BuildDynamic is a bit confusing since there is not a op called `Dynamic`, maybe something along the line of `SetDynamicDimensions`?

Reply:
That's right. I am also thinking where would be the right place to keep this method? `data_ops` is the right place still? @JackCaoG